synapse/handlers/presence.py

```diff
@@ -245,9 +250,17 @@ async def update_external_syncs_clear(self, process_id):
         """
         pass
 
-    async def process_replication_rows(self, token, rows):
+    async def process_replication_rows(
+        self, stream_name: str, instance_name: str, token: int, rows: list
+    ):
         """Process presence stream rows received over replication."""
```

Comment:
I think this docstring is now incorrect (it gets all replication rows, not just presence)?